src/schema-builder/options/VersioningOptions.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Customize temporal tables options
+ */
+export interface VersioningOptions {
```

Comment:
Maybe TemporaryTableOptions ? Versioning is very generic term and we already use that term for a different type of versioning. In this particular case it's "versioning" with "temporary table strategy".

Reply:
I change the name of this interface to `TemporalTableOptions`.